src/pages/AuthPage/AuthContainer/AuthContainerStyles.js

```diff
@@ -8,8 +8,16 @@ export const StyledAuthWrapper = styled.section`
 export const StyledAuthHeading1 = styled.h1`
   font-size: 5rem;
   margin-bottom: 1rem;
+
+  @media screen and (max-width: 500px) {
```

Comment:
this is desktop first approach for styling. But best practice is to follow mobile first approach.

Read this doc and update your styling according to mobile first approach, this PR and other PRs.
https://zellwk.com/blog/how-to-write-mobile-first-css/